test/NuGet.Core.FuncTests/Dotnet.Integration.Test/Dotnet.Integration.Test.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <ProjectReference Include="..\..\..\src\NuGet.Core\NuGet.Frameworks\NuGet.Frameworks.csproj" />
```

Comment:
I don't think this is necessary. It should already be pulled in transitively. 

this proj => NuGet.Xplat.FunctTest => NuGet.Commandline.Xplat => NuGet.Commands => NuGet.ProjectModel => NuGet.DependencyResolver.Core => NuGet.LibraryModel => NuGet.Common => NuGet.Frameworks.

Reply:
You are correct.  Validated with a fresh build both in VS and powershell.  Incoming commit to remove addition.